src/site/_filters/pretty-date.js

```diff
@@ -19,13 +19,13 @@ const {DateTime} = require('luxon');
 /**
  * Convert a JavaScript Date object into a human readable string.
  * @param {Date} date The date to convert.
- * @return {string} A human readable date string.
+ * @return {string|undefined} A human readable date string.
  */
 module.exports = (date) => {
   if (!date) {
     /* eslint-disable-next-line */
     console.warn('Date passed to prettyDate filter was undefined or null.');
-    return;
+    return undefined;
```

Comment:
just noting this in case we're allowed to call `return` without explicit `undefined`